test/integration/Platform/MysqlFixtureLoader.php

```diff
@@ -12,13 +12,14 @@
 
 class MysqlFixtureLoader implements FixtureLoader
 {
-    /** @var string */
-    private $fixtureFile = __DIR__ . '/../TestFixtures/mysql.sql';
+    private string $fixtureFile = __DIR__ . '/../TestFixtures/mysql.sql';
 
-    /** @var PDO */
-    private $pdo;
+    private PDO $pdo;
```

Comment:
These, in all of the *FixtureLoader classes must be nullable.

Reply:
I updated my other comment related to these so there is no sense having it spread into two comments. Marking this resolved.